Main application and css file

sinansevgi opened the pull request:
# Description

This pull request has a web application that renders pages dynamically.
The entire contents of the website generated using JavaScript alone. 
Non compiled Javascript files can be found in the src folder. 

## Changes
Added startserver.sh for running dev server. You can also run `npm run start:dev` to start the dev server.
All function definitions compliant with ES6 now.

sinansevgi committed:
Add github workflow and package.json

Initial project configurations

sinansevgi committed:
Add home section

sinansevgi committed:
Add contact section

sinansevgi committed:
Add navbar

sinansevgi committed:
Add menu section

sinansevgi committed:
Add License and Readme file

Srjnnnn reviewed (changes requested):
## Status: Changes Required :recycle: 

Hello Sinan :smiley: . Your page is great and all the functionality is in place. The modularity and the entrance to the app were right. However, there are just two small problems. 

- [x]  I've checked your app from the live demo and then I wanted to run my local server but there is no script in your  package.json. I can run the live server which is a built-in attribute on VScode but I think it's better to add your own scripts to run the live server. Please add this.
- [x]  Your defaults exports are using the old function syntax. Kindly change them into arrow functions. Probably you can't export in that line, so you can export at the end of the file. It is not a big deal but it is a requirement from the project specifications. Kindly change them.

After making those changes please submit for a re-review.
If you have any questions about the review, feel free to reach out to me via Slack. :100: 
_Eyüp Sercan UYGUR_

sinansevgi committed:
Fix function definitions.
Add run script for live server.

Changed old function definitions with ES6 definitions.
Added configuration and script for webpack dev server.

uche-inyama reviewed (approved):
## Approved

### Hi @sinansevgi :wave: 

Go on with the next task.

However, in your readme, update the instruction on how to run the application on your local machine, to include: running **npm install** then **npm run start:dev** to start the start application.

### Have a great day :slightly_smiling_face: 

#### ``` Uchechukwu Henry Inyama ```

sinansevgi committed:
Update README.md

sinansevgi committed:
Update README.md

sinansevgi committed:
Merge remote-tracking branch 'origin/main_project' into main_project

# Conflicts:
#	README.md
